Readme format (#2179)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Added a new validation to the **validate** command to verify that the docker in use is not deprecated.
* Added support for multiple ApiModules in the **unify** command

> * Added validation to **validate** command to identify unsupported relative urls in README files.

Reviewer: ```suggestion
* Added a check to **validate** command, preventing use of relative urls in README files.
```

* Added a new validation to the **validate** command to verify that the docker in use is not deprecated.
* Added support for multiple ApiModules in the **unify** command
* Added slack notifier for build failures on the master branch.

> * Added validation to **validate** command to identify unsupported relative urls in README files.

Reviewer 2: Missing a note abut foramt

Author: Added it 👍

* Locked the dependency on Docker.
* Removed a traceback line from the **init** command templates: BaseIntegration, BaseScript.
* Updated the token in **_add_pr_comment** method from the content-bot token to the xsoar-bot token.
* Added a new validation to the **validate** command to verify that the docker in use is not deprecated.

> * Added support for multiple ApiModules in the **unify** command

Reviewer: please only add to `Unreleased`
